package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "homepage": "https://elena-myone.github.io/my-best-show",
+  "name": "my-best-show",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

Comment:
![4](https://github.com/Elena-MyOne/my-best-show/assets/74279859/a34e1f31-08c3-4812-a7f5-756556b22c66)

I'm usually using this panel and don't start scripts through the terminal, it's more convenient from my perspective. I rename from `dev` into `start` but will not add `--start` if you don't mind. I know about this flag  and don't use it, because I will not have to close the window if I add dependencies